specifications/api/swagger.yaml

```diff
@@ -848,6 +848,7 @@ components:
         decentralization_level: *percentage
         desired_pool_number: *stakePoolsNumber
         minimum_utxo_value: *amount
+        hardfork_epoch_info: *epochInfo
```

Comment:
What about just `hardfork` ? or `hardfork_at` for consistency across the API namings?

Reply:
ok, will change very soon

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1932/commits/ed5a88542614083270857c69a9cb44e5515ce9b1